section2/reflection.md

```diff
@@ -2,14 +2,43 @@
 
+1) Don't panic when you encounter struggle. Observe your thoughts/emotions and find a way forward.
+2) Appreciate the progress you've made rather than looking at how far you have to go.
+
+
 2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
 
+I am embracing the grind and commitment, knowing it'll get me to where I want to be. I am excited to push myself to learn new things.
+
+I let imposter syndrome cloud my thoughts too often. I need to acknowledge that everyone starts from zero and with time and practice, I will have a standard of work I can look at as professional.
+
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
 
+I feel comfortable with the nuance of most of the processes we have learned so far. I don't find noding a file or establishing a local repo/git repo relationship inherently confusing. Getting practice with the exercises has also reinforced those processes.
+
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
 
+Essentially, we are telling the computer to look at a variable. If that variable meets a certain condition, then the computer knows to respond in a certain fashion. If the value of the variable does not match the condition, then the computer knows to respond in a different fashion.
+
+Ex. for website associated with the sale of alcohol, you must enter your birthday before entering. If your age is listed shows you are over 21, you may continue. If your age is below 21, the site kicks you back to the browser.
+
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
 
+We use else. The computer evaluates one condition at a a time
+
+If (condition) then -> console log
+else (condition) then -> console log
+else (condition) then -> console log
+else -> console log
+
+the computer will keep trying to satisfy the conditions in order, and will keep moving down the tree until it finds a condition the value satisfies
+
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+Logical operators!
+
+&& === and (to check if both conditions are met)
+|| === or (to check if one or the other are met)
+
+7. What questions do you still have about `if` statements and/or functions?
+Why do curly brackets {} have to terminate on the next line down after the code function instead of on the end of the previous line?
```

Comment:
This makes it more readable for other engineers to know where everything ends when scrolling through code.